dev-libs/benchmark/benchmark-1.3.0.recipe

```diff
@@ -0,0 +1,70 @@
+SUMMARY="A microbenchmark support library"
+DESCRIPTION="A library to support the benchmarking of functions, similar to unit-tests."
+HOMEPAGE="https://github.com/google/benchmark"
+COPYRIGHT="2015-2017 Google Benchmark developers"
```

Comment:
As explained in [`CONTRIBUTORS`][0], developing the library does not implict copyright. Copyright holders instead are listed in [`AUTHORS`][0]. So I suggest we switch the wording here.

```diff
-COPYRIGHT="2015-2017 Google Benchmark developers"
+COPYRIGHT="2015-2017 Google Benchmark authors"
```

[0]: https://github.com/google/benchmark/blob/v1.3.0/CONTRIBUTORS#L3
[1]: https://github.com/google/benchmark/blob/v1.3.0/AUTHORS